vertical-pod-autoscaler/pkg/updater/eviction/pods_eviction_restriction.go

```diff
@@ -203,10 +206,15 @@ func (f *podsEvictionRestrictionFactoryImpl) NewPodsEvictionRestriction(pods []*
 	creatorToSingleGroupStatsMap := make(map[podReplicaCreator]singleGroupStats)
 
 	for creator, replicas := range livePods {
+		// Use per-VPA minReplicas if present, fall back to the global setting.
```

Comment:
I don't think we should do this in every iteration of the loop

Reply:
True, fixed.